### ISSUE TITLE
The first task in the included tasks yml using ansible.builtin.include_tasks shows up ansible lint issue is "not a valid attribute for a Play"

### ISSUE THREAD

**Issue by sachin-apa** [ansible/ansible-lint#3209]
##### Summary

When we use include_tasks in a loop the play in the included file is reporting the ansible-lint issues as below

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

```console (paste below)
# ansible-lint --version
ansible-lint 6.14.2 using ansible 2.14.3
```

Running on Red Hat Enterprise Linux release 9.1 (Plow) with a latest version of `ansible`, `ansible-lint` and `python 3.9`
- ansible installation method: one of source, pip
- ansible-lint installation method: pip

##### STEPS TO REPRODUCE
Sample playbooks for the execution is as below.
Create a playbook for a main task `main_task.yml` and a sub task `sub_task.yml` as below (These are created to replicate issue, the real world scenario is different)
```console (paste below)
---
# ./main_task.yml
- name: Say Hello
  hosts: "*"
  gather_facts: false
  tasks:
    - name: Call hello message
      ansible.builtin.include_tasks: sub_task.yml
      loop:
        - World
        - Earth
      loop_control:
        loop_var: name
        index_var: index
```
```console (paste below)
---
# ./sub_task.yml
- name: Construct hello msg
  ansible.builtin.set_fact:
    greeting: "Greeting {{ index }}: Hello, {{ name }}!"

- name: Print hello msg
  ansible.builtin.debug:
    msg: "{{ greeting }}"
```
##### Desired Behavior

Expected that ansible-lint will not through any issues.

##### Actual Behavior

Running an `ansible-lint` is showing the following error:
```console
# ansible-lint ./myscenario
WARNING  Listing 1 violation(s) that are fatal
syntax-check[specific]: 'ansible.builtin.set_fact' is not a valid attribute for a Play
myscenario/sub_task.yml:3:3


                  Rule Violation Summary                   
 count tag                    profile rule associated tags 
     1 syntax-check[specific] min     core, unskippable    

Failed after : 1 failure(s), 0 warning(s) on 3 files.
```
```
# ansible-playbook --syntax-check ./roles/idrac_gather_facts/molecule/myscenario/sub_task.yml 
[WARNING]: No inventory was parsed, only implicit localhost is available
[WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'
ERROR! 'ansible.builtin.set_fact' is not a valid attribute for a Play

The error appears to be in '/root/ansible-internal/Conn-Ansible/roles/idrac_gather_facts/molecule/myscenario/sub_task.yml': line 3, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

# ./sub_task.yml
- name: Construct hello msg
  ^ here
```
Here cannot include hosts, tasks etc. as this are all part of the main_task.yml file.
Please let me know if there is any way we can resolve this lint issue.

**Comment by ajinkyau** [comment]
Hi @sachin-apa,

We can see that the task file is not present in the tasks folder which should fix this issue.
Also, you cannot pass a task file to syntax-check

**Comment by sachin-apa** [comment]
Sorry, 

> Hi @sachin-apa,
> 
> We can see that the task file is not present in the tasks folder which should fix this issue. Also, you cannot pass a task file to syntax-check

Sorry didn't understood the reason for closing this issue.
Also, task file is not present in the tasks folder, Can you please explain what do you mean by this?
